safearea/src/safearea_null.cpp

```diff
@@ -1,11 +1,11 @@
 #include "safearea.h"
 
-#ifndef DM_PLATFORM_IOS
+#ifndef DM_PLATFORM_IOS && DM_PLATFORM_ANDROID
```

Comment:
Is this working? I've learned that for multiple define checks you need to use defined() like this::

`#if !defined(DM_PLATFORM_IOS) && !defined(DM_PLATFORM_ANDROID)`

Reply:
Good point, thank you!